editor/src/core/property-controls/property-controls-parser.ts

```diff
@@ -95,7 +95,7 @@ export function parseNumberInputControlDescription(
       return numberInputControlDescription
     },
     optionalObjectKeyParser(parseString, 'label')(value),
-    objectKeyParser(parseEnum(['number-input']), 'control')(value),
+    objectKeyParser(parseConstant('number-input'), 'control')(value),
```

Comment:
`parseEnum` 's typing can only guarantee that the result is a string and not a 'number-input'